### PR TITLE
 +Theken-Floalts  +Batteriestände Fernbedinungen

### PR DESCRIPTION
**Pull request by Ranlvor**

### CODE REVIEW

**Comment by uwap** [comment]
@Ranlvor This has open conflicts that need to be resolved

**Comment by Ranlvor** [comment]
Conflict resolved.

File: config/rzl.js [hunk 1]
```diff
@@ -670,7 +680,99 @@ const config : Config = {
...
+          delayedApply: true
+        }
+      ]
+    },
+    remotes: {
+      name: "Fernbedinungen",
+      position: [400, 348],
+      icon: "remote",
+      iconColor: (state) =>
+        ((state[tradfri_remote.low("65536")] == "true") || (state[tradfri_remote.low("65542")] == "true") || (state[tradfri_remote.low("65546")] == "true") || (state[tradfri_remote.low("65547")] == "true")) ? hex("#ff0000") : hex("#000000"),
```
Comment (uwap) [inline config/rzl.js]: Please refactor! Drop the unneeded parenthesis. Also this line is way to long. Please break it into multiple lines.

You could also use the [_.every](https://lodash.com/docs/4.17.10#every) function of lodash to increase readability.

File: config/utils.js [hunk 2]
```diff
@@ -59,6 +59,25 @@ export const floalt = {
   })
 }
 
+export const tradfri_remote = {
+  level: (remote_id: string) => `tradfri_remote_${remote_id}_level`,
+  low: (remote_id: string) => `tradfri_remote_${remote_id}_low`,
+  topics: (remote_id: string) => ({
+    [ `tradfri_remote_${remote_id}_level` ]: {
+      state: `/service/openhab/out/tradfri_0830_gwb8d7af2b448f_${remote_id}_battery_level/state`,
+      command: ``,
```
Comment (uwap) [inline config/utils.js]: Please don't use unneeded template strings here

File: config/utils.js [hunk 3]
```diff
@@ -59,6 +59,25 @@ export const floalt = {
...
+      defaultValue: "0",
+      values: {}
+    },
+    [ `tradfri_remote_${remote_id}_low` ]: {
+      state: `/service/openhab/out/tradfri_0830_gwb8d7af2b448f_${remote_id}_battery_low/state`,
+      command: ``,
```
Comment (uwap) [inline config/utils.js]: And here

**Pull request merged**